pages/compare.py

```diff
@@ -160,8 +194,13 @@ def get_user():
 def create_prompt_model(model_id, prompt, position):
   if position not in ["PREFIX", "SUFFIX", "TEMPLATE"]:
     raise Exception("Position must be PREFIX or SUFFIX")
+  
+  validate_scopes(['Apps_Get','Concepts_Get','Models_Add','Models_Train','Models_Get','Workflows_Get'], myscopes_response.scopes, "PostModels")
```

Comment:
nice this is great, I was thinking we probably want to call validate_scopes once and at the top after we get the ClarifaiAuthHelper created rather than each request so that it fails fast. In a future PR we can move the vlidate_scopes method onto that ClarifaiAuthHelper so that every module can use it from the python utils.